docs/api-inference/tasks/chat_completion.md

```diff
@@ -0,0 +1,203 @@
+## Chat completion
+
+Generate a response given a list of messages.
+This is a subtask of [`text-generation`](./text_generation) designed to generate responses in a conversational context.
+
+
+
+### Recommended models
+
+- [google/gemma-2-2b-it](https://huggingface.co/google/gemma-2-2b-it): A text-generation model trained to follow instructions.
+- [meta-llama/Meta-Llama-3.1-8B-Instruct](https://huggingface.co/meta-llama/Meta-Llama-3.1-8B-Instruct): Very powerful text generation model trained to follow instructions.
+- [microsoft/Phi-3-mini-4k-instruct](https://huggingface.co/microsoft/Phi-3-mini-4k-instruct): Small yet powerful text generation model.
+- [AI-MO/NuminaMath-7B-TIR](https://huggingface.co/AI-MO/NuminaMath-7B-TIR): A very powerful model that can solve mathematical problems.
```

Comment:
As discussed in a follow-up, we should prevent having frozen models here

Reply:
yes, related thread: https://github.com/huggingface/hub-docs/pull/1384#discussion_r1729042102

Reply:
https://github.com/huggingface/hub-docs/pull/1396/files